cmd/terramate/cli/cli.go

```diff
@@ -200,21 +220,34 @@ func newCLI(
 		}
 	}
 
+	logger := log.With().
+		Str("action", "newCli()").
+		Str("stack", wd).
+		Logger()
+
+	logger.Trace().
+		Msgf("Evaluate symbolic links for %q.", wd)
 	wd, err = filepath.EvalSymlinks(wd)
 	if err != nil {
 		return nil, fmt.Errorf("failed evaluating symlinks for %q: %w", wd, err)
 	}
 
+	logger.Trace().
+		Msgf("Get absolute file path of %q.", wd)
 	wd, err = filepath.Abs(wd)
 	if err != nil {
 		return nil, fmt.Errorf("getting absolute path of %q: %w", wd, err)
 	}
 
+	logger.Trace().
+		Msgf("Change working directory to %q.", wd)
```

Comment:
```suggestion
		Msg("Change process working directory")
```

Rationale: avoid formatting in favor of fields + we already have the working dir value on the logger context (must be fixed from "stack").